### ISSUE TITLE
A11y: Mobile view: Service pop-up in the footer is incorrectly structured and cannot be operated via keyboard and screen reader

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7083]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The ‘Service’ collapse in the footer with a narrow viewport display cannot be operated using keyboard or screen reader navigation, as a simple div HTML element is used for this, which has been labelled with the correct, additional aria attributes, but still has the wrong role.

*_Expected behaviour:_*

Set the ‘Service’ pop-up according to the ARIA disclosure pattern.
https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag:
- open a page with a mobile viewport and scroll down to the page footer

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40839**



[created from NEXT-40839, comment 541775]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40839, comment 542006]

**Comment by shopwareBot** [comment]
[public] Thank you so much for your detailed issue report and supporting us to make the internet more accessible! This is already covered by the following issue: [https://github.com/shopware/shopware/issues/6995](https://github.com/shopware/shopware/issues/6995)



[created from NEXT-40839, comment 542224]
